test/us_state_abbrev_test.py

```diff
@@ -0,0 +1,17 @@
+from libs.us_state_abbrev import (
+    US_STATE_ABBREV,
+    abbrev_us_state,
+    ABBREV_US_FIPS,
+    ABBREV_US_UNKNOWN_COUNTY_FIPS,
+)
+
+
+def test():
```

Comment:
Could you give this test a slightly more descriptive name?

Reply:
Ha, could hardly be less descriptive :-P Done.